keep/providers/graylog_provider/graylog_provider.py

```diff
@@ -526,7 +526,7 @@ def __map_event_to_alert(event: dict) -> AlertDto:
         return alert
 
     @staticmethod
-    def _format_alert(event: dict, provider_instance: BaseProvider) -> AlertDto:
+    def _format_alert(event: dict, provider_instance: BaseProvider = None) -> AlertDto:
```

Comment:
```suggestion
    def _format_alert(event: dict, provider_instance: BaseProvider | None = None) -> AlertDto:
```
is it failing because of it?